integrated_channels/sap_success_factors/transmitters/content_metadata.py

```diff
@@ -62,17 +67,17 @@ def transmit(self, payload, **kwargs):
                 # Remove the failed items from the create/update/delete dictionaries,
                 # so ContentMetadataItemTransmission objects are not synchronized for
                 # these items below.
-                self._remove_failed_items(chunked_items, items_to_create, items_to_update, items_to_delete)
+                self._remove_failed_items(chunked_items, create_payload, items_to_update, items_to_delete)
 
         # If API transmission limit is set then mark the rest of the items as not transmitted.
         # Since, chunk_items is a generator and we have already iterated through the items that need to
         # be transmitted. Rest of the items are the ones that need to marked as not transmitted.
         for chunk in chunk_items:
             chunked_items = list(chunk.values())
-            self._remove_failed_items(chunked_items, items_to_create, items_to_update, items_to_delete)
+            self._remove_failed_items(chunked_items, create_payload, items_to_update, items_to_delete)
 
-        self._create_transmissions(items_to_create, items_catalog_updated_times, content_catalog_map)
-        self._update_transmissions(items_to_update, transmission_map, items_catalog_updated_times)
+        self._create_transmissions(create_payload, content_updated_mapping)
```

Comment:
I think having the ordering be consistent between the methods would be great, for ex the content_update_mapping being last or sth like that